SwordImportExportPlugin.inc.php

```diff
@@ -191,18 +194,22 @@ public function display($args, $request) {
 
 				$errors = array();
 				// select at least one article
-				$articleIds = $request->getUserVar('articleId');
-				if (empty($articleIds)) {
+				$submissionIds = $request->getUserVar('submissionId');
```

Comment:
Shouldn't this be `array($request->getUserVar('submissionId'))` to avoid a warning?